scripts/create_burndown.py

```diff
@@ -17,211 +17,402 @@
-  new_fast_lane_cards = [0]
-  for i in range(1, len(total_fast_lane)):
-    new_fast_lane_cards.append(total_fast_lane[i] - total_fast_lane[i - 1])
-  effective_new_fast_lane_days = []
-  effective_new_fast_lane_cards = []
-  for i in range(len(new_fast_lane_cards)):
-    if new_fast_lane_cards[i] > 0:
-      effective_new_fast_lane_days.append(i + 0.1 + 1)
-      effective_new_fast_lane_cards.append(new_fast_lane_cards[i])
-  if len(effective_new_fast_lane_cards) > 0:
-    plt.bar(effective_new_fast_lane_days, effective_new_fast_lane_cards, .2, color='red')
-
-# Draw arrow showing already done tasks at begin of sprint
-tasks_done = burndown["days"][0]["tasks"]["total"] - burndown["days"][0]["tasks"]["open"]
-
-if tasks_done > 0 and not args.no_tasks:
-  plt.annotate("",
-	       xy=(x_days[0], scalefactor * max_story_points - 0.5 ), xycoords='data',
-	       xytext=(x_days[0], y_open_tasks[0] + 0.5), textcoords='data',
-	       arrowprops=dict(arrowstyle="<|-|>", connectionstyle="arc3", color='green')
-	      )
-
-  y_text = scalefactor * max_story_points
-  plt.text(0.7, y_text, str(int(tasks_done)) + " tasks done",
-	   rotation='vertical', verticalalignment='top', color='green'
-	  )
-
-if burndown["days"][0].has_key("tasks_extra") and not args.no_tasks:
-  tasks_done_extra = burndown["days"][0]["tasks_extra"]["done"]
-
-  plt.annotate("",
-	       xy=(x_days[0], scalefactor * y_story_points_done_extra[0] - 0.5 ), xycoords='data',
-	       xytext=(x_days[0], 0.5 - tasks_done_extra), textcoords='data',
-	       arrowprops=dict(arrowstyle="<|-|>", connectionstyle="arc3", color='green')
-	      )
-
-  y_text = - tasks_done_extra / 2
-  plt.text(0.4, y_text, str(int(tasks_done_extra)) + " extra",
-	   rotation='vertical', verticalalignment='center', color='green'
-	  )
-  plt.text(0.7, y_text, "tasks done",
-	   rotation='vertical', verticalalignment='center', color='green'
-	  )
 
+# Parse arguments and/or print help
+def argParser():
+  epilog = "Look at https://github.com/openSUSE/trollolo for details"
+  description = "Generates Scrum Burndown Chart from YAML file"
+  
+  parser = argparse.ArgumentParser(epilog=epilog, description=description)
+  parser.add_argument('sprint', metavar='NUM', help='Sprint Number')
+  parser.add_argument('--output', help='Location of data to process')
+  parser.add_argument('--no-tasks', action='store_true', help='Disable Tasks line in the chart', default=False)
+  parser.add_argument('--with-fast-lane', action='store_true', help='Draw line for Fast Lane cards', default=False)
+  parser.add_argument('--verbose', action='store_true', help='Verbose Output', default=False)
+  parser.add_argument('--no-head', action='store_true', help='Run in headless mode', default=False)
+  args = parser.parse_args()
+  
+  if args.output:
+    os.chdir(args.output)
+  
+  if args.verbose:
+    print args
+
+  return args
+
+
+
+class BurndownData:
+  "Store burndown data parsed from YAML file"
+
+  # Constructor
```

Comment:
This comment is redundant. In Python it's clear that `__init__` is the constructor.